\chapter{Main Findings} % Consider renaming to abstract

What did I do so far?

programming and data
\begin{itemize}
	\item Custom Analysis pipelines
	\item Evaluation of \foreignword{de novo} Assembly Software with 454 data
	\item Evaluation of Assembly Software with Illumina data
	\item Chimeric contig detection
	\item Comparison of public datasets from different sources by complexity reduction
\end{itemize}

metabolism
\begin{itemize}
	\item \species{Megathyrsus maximus} PEP-CK type C$_4$
	\item \species{Megathyrsus} blueprint for engineering C$_4$-cycle
\end{itemize}
	
transport
\begin{itemize}
	\item \species{Megathyrsus} intercellular transport requirement
	\item \species{Megathyrsus} modular intracellular transport machinery
\end{itemize}



%%%PLAYGROUND. To be deleted in the final version
\subsection{Custom analysis pipelines}
this one is very generic so I will fit it in somewhere in between
\subsection{Evaluation of assembly softwares}
For de novo assembly of 454 pyrosequencing reads we tested six different assembly algorithms with simulated reads. These reads were extracted from the \species{Arabidopsis} genome and were therefore considered as perfect reads. To get a more realistic picture of assembly we additionally modified the reads with 1\%/3\%/5\% \foreignword{in silico} base changes.
The six assembly algorithms, mira \cite{unknown}, velvet \cite{unknown}, SOAP \cite{unknown}, CAP3 \cite{unknown}, TGICL \cite{unknown}, and CLC \foreignword{de novo} assembly \cite{unknown}, qualitatively performed similar, with contig numbers in the $10^5$s and N50s between 476 and 732.

% Starting over again with more focus on the FINDINGS
\subsubsection{Critical Assessment of Assembly Strategies
\cite{mp_Braeutigam2011}}

In this study, we tested six assembly algorithms\footnote{Graph-based: SOAP, Velvet, MIRA; OLC-based: CAP3, TGICL; proprietary: CLC} for quality in \foreignword{de novo} assembly of 454 data. We could show that CAP3 and TGICL are more robust against point mutations which simulated sequencing error, as well as biological variance.
Furthermore, we showed that the tested graph-based algorithms have difficulties assembling full-length transcripts, even when a high number of reads is available. In contrast, the OLC-based assemblers and the proprietary algorithm by CLCbio produced mostly full-length transcripts read number above 100.

%%%