\chapter{Conclusion}
 \section{Next-Generation Sequencing}
	A variety of sequencing platforms, software, and use-cases is present in sequencing experiments.
	In our study, we wanted to evaluate the transcriptomes of non-model species (i.e. species without a sequenced genome).
	At the time of the initial experimental design, 454 sequencing was the commonly favorited approach for sequencing non-model species.
	However, it remained unclear, whether \ac{OLC} or \ac{DBG} assemblers were more successful in assembling full-length contigs for each transcript.
	To find an answer to this question, we used the \species{Arabidopsis thaliana} genome as a well known reference.
	From this reference we generated simulated 454 reads, based on actual sequencing data from former studies in \species{Cleome} \cite{Braeutigam2010}.
	These simulated reads were perfect in terms of sequencing errors.
	Therefore, we additionally created read libraries with artificial 1\%, 3\%, and 5\% base changes.
	With the 4 read libraries as input sequences, we used the assembly software: SOAP\cite{unknown}, Velvet\cite{unknown}, MIRA\cite{unknown}, CAP3\cite{unknown}, TGICL\cite{unknown}, and \cite{CLC}.
	%Time to go home for today. Tomorrow I will continue with how to decide on the quality of all the algorithms
	Traditional quality parameters ...  did not allow for decision -> better than 